### ISSUE TITLE
Compatible with EMQX broker

### ISSUE THREAD

**Issue by alvin1221** [wolfSSL/wolfMQTT#271]
Hi there,
     We have tested that wolfMQTT client connect to **emqx broker** successfully, also publish & subscribe message are working fine. I hope **EMQX broker** can be added to the **broker compatibility** list of **README.md**, thanks !

MQTT v3.1.1
```bash
./examples/mqttclient/mqttclient -h "broker.emqx.io" 
MQTT Client: QoS 0, Use TLS 0
MQTT Net Init: Success (0)
MQTT Init: Success (0)
NetConnect: Host broker.emqx.io, Port 1883, Timeout 5000 ms, Use TLS 0
MQTT Socket Connect: Success (0)
MQTT Connect: Proto (v3.1.1), Success (0)
MQTT Connect Ack: Return Code 0, Session Present 0
MQTT Subscribe: Success (0)
  Topic wolfMQTT/example/testTopic, Qos 0, Return Code 0
MQTT Publish: Topic wolfMQTT/example/testTopic, Success (0)
MQTT Waiting for message...
MQTT Message: Topic wolfMQTT/example/testTopic, Qos 0, Len 4
Payload (0 - 4): test
MQTT Message: Done
```

MQTT v5
```bash
./examples/mqttclient/mqttclient -h "broker.emqx.io"
MQTT Client: QoS 0, Use TLS 0
MQTT Net Init: Success (0)
MQTT Init: Success (0)
NetConnect: Host broker.emqx.io, Port 1883, Timeout 5000 ms, Use TLS 0
MQTT Socket Connect: Success (0)
MQTT Connect: Proto (v5), Success (0)
MQTT Connect Ack: Return Code 0, Session Present 0
MQTT Connect Ack: Assigned Client ID: WolfMQTTClient
MQTT Subscribe: Success (0)
  Topic wolfMQTT/example/testTopic, Qos 0, Return Code 0
MQTT Publish: Topic wolfMQTT/example/testTopic, Success (0)
MQTT Waiting for message...
MQTT Message: Topic wolfMQTT/example/testTopic, Qos 0, Len 4
Payload (0 - 4): test
MQTT Message: Done
```

**Comment by alvin1221** [comment]
> Hi there, We have tested that wolfMQTT client connect to **emqx broker** successfully, also publish & subscribe message are working fine. I hope **EMQX broker** can be added to the **broker compatibility** list of **README.md**, thanks !
> 
> MQTT v3.1.1
> 
> ```shell
> ./examples/mqttclient/mqttclient -h "broker.emqx.io" 
> MQTT Client: QoS 0, Use TLS 0
> MQTT Net Init: Success (0)
> MQTT Init: Success (0)
> NetConnect: Host broker.emqx.io, Port 1883, Timeout 5000 ms, Use TLS 0
> MQTT Socket Connect: Success (0)
> MQTT Connect: Proto (v3.1.1), Success (0)
> MQTT Connect Ack: Return Code 0, Session Present 0
> MQTT Subscribe: Success (0)
>   Topic wolfMQTT/example/testTopic, Qos 0, Return Code 0
> MQTT Publish: Topic wolfMQTT/example/testTopic, Success (0)
> MQTT Waiting for message...
> MQTT Message: Topic wolfMQTT/example/testTopic, Qos 0, Len 4
> Payload (0 - 4): test
> MQTT Message: Done
> ```
> 
> MQTT v5
> 
> ```shell
> ./examples/mqttclient/mqttclient -h "broker.emqx.io"
> MQTT Client: QoS 0, Use TLS 0
> MQTT Net Init: Success (0)
> MQTT Init: Success (0)
> NetConnect: Host broker.emqx.io, Port 1883, Timeout 5000 ms, Use TLS 0
> MQTT Socket Connect: Success (0)
> MQTT Connect: Proto (v5), Success (0)
> MQTT Connect Ack: Return Code 0, Session Present 0
> MQTT Connect Ack: Assigned Client ID: WolfMQTTClient
> MQTT Subscribe: Success (0)
>   Topic wolfMQTT/example/testTopic, Qos 0, Return Code 0
> MQTT Publish: Topic wolfMQTT/example/testTopic, Success (0)
> MQTT Waiting for message...
> MQTT Message: Topic wolfMQTT/example/testTopic, Qos 0, Len 4
> Payload (0 - 4): test
> MQTT Message: Done
> ```

**Comment by embhorn** [comment]
Perfect, thank you @alvin1221 !

**Comment by embhorn** [comment]
Added to readme. 
Fixed by #297
